SPRCLI/Sources/SPR/UploadArchive.swift

```diff
@@ -50,7 +50,7 @@ extension SPRPublisher {
         let fileHandle = try FileHandle(forReadingFrom: archiveFileURL)
         let stream = FileStream(fileHandle: fileHandle)
         let body = ByteStream.stream(stream)
-        let input = PutObjectInput(body: body, bucket: bucket, contentType: "application/zip", key: archiveKey)
+        let input = PutObjectInput(body: body, bucket: bucket, cacheControl: "public, immutable", contentType: "application/zip", key: archiveKey)
```

Comment:
Archives should never, ever change once published so this cache-control is set to immutable.